src/catch2/internal/catch_console_colour.cpp

```diff
@@ -115,10 +114,13 @@ namespace {
         }
 
         static bool useImplementationForStream(IStream const& stream) {
```

Comment:
We might just kick this one out of the auto-selection. Even the version check here is wonky, because it's easily thrown off guard by manifests or compatibility mode.